wiki/People/en.md

```diff
@@ -4,6 +4,8 @@ The following are user groups which are made up of the osu! community members th
 
 - [Beatmap Nominators](/wiki/People/Beatmap_Nominators)
 - [Community Contributors](/wiki/People/Community_Contributors)
+<!-- This should be its own page under People -->
```

Comment:
maybe at this in the same line as Developers?

Reply:
does it matter it's just a comment 🤔

Reply:
thats why making a seperate line just for it seems unnecessary 🤷‍♂